www/pycd01/cv03/index.php

```diff
@@ -0,0 +1,115 @@
+
+ <?php
+
+ $errors = [];
+ $successMess = "";
+
+ if(!empty($_POST))
+ {
+     $name = htmlspecialchars(trim($_POST['name'], ' \t'));
```

Comment:
proc je tady jenom `\t`? https://www.php.net/manual/en/function.trim.php

Reply:
"\r", "\n" a "\v" není možné vložit do single-line inputu, tak jsem se rozhodl je vynechat. Zjišťoval jsem si něco o null bytech a nevědel jsem, že je s nima možné udělat např. SQL injection u starších DB.